app/views/home_page/index.html.slim

```diff
@@ -0,0 +1,4 @@
+h1
+  | Hello World!
```

Comment:
Please use localization for text in views. Let's store these strings in the `en.yml`